replicator/src/main.rs

```diff
@@ -42,7 +42,7 @@ fn main() {
         .arg(
             Arg::with_name("storage_keypair")
                 .short("s")
-                .long("storage_id")
+                .long("storage-keypair")
                 .value_name("DIR")
```

Comment:
nit: might as well update `DIR` while you're here

Reply:
Still a DIR though :)